plugins/generator-1.21.1/neoforge-1.21.1/procedures/entity_get_attribute_base_value.java.ftl

```diff
@@ -0,0 +1 @@
+(${input$entity} instanceof LivingEntity livingEntity${cbi} && livingEntity${cbi}.getAttributes().hasAttribute(${generator.map(field$attribute, "attributes")}) ? livingEntity${cbi}.getAttribute(${generator.map(field$attribute, "attributes")}).getBaseValue() : 0)
```

Comment:
All tmp variables in MCreator in block code should start with an underscore (_), as we don't allow local variables to start with that name. Otherwise, local variable called e.g. "livingEntity0" will cause name collision.

Now with these checks in place, it may also make sense to split those into more than one line for the sake of readability.

Example (of line split and underscore name):

![image](https://github.com/user-attachments/assets/ccc49266-4d62-455d-bfb6-0ef653151b57)